assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js

```diff
@@ -31,15 +31,18 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
 import { CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
-import { FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
+import { DATE_RANGE_OFFSET, FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
 import GoogleChart from '../../../../../components/GoogleChart';
 import parseDimensionStringToDate from '../../../util/parseDimensionStringToDate';
 import PreviewBlock from '../../../../../components/PreviewBlock';
 import ReportError from '../../../../../components/ReportError';
 const { useSelect } = Data;
 
 export default function UserCountGraph( { loaded, error, report } ) {
+	const { startDate, endDate } = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( { offsetDays: DATE_RANGE_OFFSET } ) );
+	const numberOfDays = useSelect( ( select ) => select( CORE_USER ).getDateRangeNumberOfDays() );
```

Comment:
Nitpicking a bit, but we should clarify that this comes from the date range.
```suggestion
	const dateRangeNumberOfDays = useSelect( ( select ) => select( CORE_USER ).getDateRangeNumberOfDays() );
```

As a side note – while this selector is likely useful and good to have, couldn't we base this on the number of rows as well? I would expect them to be the same, maybe +/- 1 for headers.